js/packages/phoenix-client/package.json

```diff
@@ -4,8 +4,13 @@
   "description": "A client for the Phoenix API",
   "main": "dist/src/index.js",
   "module": "dist/esm/index.js",
-  "esnext": "dist/esnext/index.js",
   "types": "dist/src/index.d.ts",
+  "exports": {
+    ".": {
+      "import": "./dist/esm/index.js",
+      "require": "./dist/src/index.js"
```

Comment:
so tsc-alias re-maps this block post build is that how that works?

Reply:
It rewrites the actual import statements in all code inside the `dist/esm` folder.

We write code like `import thing from "./my-code"` but that is not valid ESM, you need file extensions like `import thing from ".my-code.js"`. `ts-alias` will add all of those extensions to relative imports in a second build step, after `tsc` completes